src/cloud-api-adaptor/ibmcloud-powervs/image/copy-files.sh

```diff
@@ -6,27 +6,44 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 PODVM_DIR=${REPO_ROOT}/podvm
 
 sudo mkdir -p /etc/containers
-sudo cp /tmp/files/etc/aa-offline_fs_kbc-keys.json /etc/aa-offline_fs_kbc-keys.json
+sudo cp "${PODVM_DIR}"/files/etc/agent-config.toml /etc/agent-config.toml
+sudo cp "${PODVM_DIR}"/files/etc/aa-offline_fs_kbc-keys.json /etc/aa-offline_fs_kbc-keys.json
```

Comment:
Well IMO quoting the whole string looks nicer (and avoids issues with spaces, if any), but not really a big issue. Maybe consider for the next time...